doc/private-topics.rst

```diff
@@ -0,0 +1,76 @@
+##################
+Non XML SAL Topics
+##################
+
+Private Fields
+==============
+
+private_sndStamp (TAI, unix seconds)
+    Time at which the sample was sent.
+private_rcvStamp (TAI, unix seconds)
+    Time at which sample was received.
+private_identity
+    For a CSC this has the form <SAL component name>[:index] e.g. "Script:5" or "ATDome".
+    For a user this has the form username@host.
+    This field is used for authorization.
+    New in ts_sal 4.2.
+private_origin
+    The process ID of the writer.
+private_host
+    The host address of the writer.
+    This is deprecated, and due to be removed in ts_sal 5.
+private_seqNum
+    For commands this uniquely identifies the command (within a reasonable timespan, since the value must eventually wrap around).
+    For other kinds of topics this may be incremented for each sample, though that is optional.
+<SAL component name>ID (ATDomeID)
+    Only present for indexed SAL components.
+    The SAL index of the writer.
+
+Non XML Topics
+==============
+
+ackcmd
+------
+
+private_seqNum
+    The private_seqNum of the command being acknowledged.
+ack
+    The acknowledgement code, one of CMD\_ constants, such as CMD_COMPLETE = 303 or CMD_FAILED = -302.
+error
+    The error code.
+    Set to a nonzero value if ack=CMD_FAILED.
+result
+    Salobj sets this to an error message if ack=CMD_FAILED.
+identity
+    The private_identity of the command being acknowledged (see below).
```

Comment:
Instead of "see below" please replace this with a link to the info -- which is above, since you describe the private fields before you list the ackcmd.

Specifically I suggest making `private_identity` a link and eliminating the text " (see below)".

Reply:
Likewise please consider making all the private_xxx text in the descriptions of ackcmd link to the description of that private field.